source/javascripts/components/GuidedOnboarding/Collapsible/Collapsible.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { useState, useRef, useEffect } from "react";
+import { Icon, Text, Flex } from "@bitrise/bitkit";
+import "./Collapsible.scss";
+interface CollapsibleProps {
+  open?: boolean;
+  title: string;
+}
+
+const Collapsible: React.FC<CollapsibleProps> = ({ open = false, children, title }) => {
+  const [isOpen, setIsOpen] = useState(open);
+  const [contentHeight, setContentHeight] = useState<number | undefined>(
+    open ? undefined : 0
+  );
+  const ref = useRef<HTMLDivElement>(null);
+
+  useEffect(() => {
+    if (isOpen) setContentHeight(ref.current?.getBoundingClientRect().height);
+    else setContentHeight(0);
+  }, [isOpen]);
+
+  const toggleOpen = () => {
+    setIsOpen((prev) => !prev);
+  };
+
+  return (
+      <Flex
```

Comment:
I know WFE is lacking the latest BitKit features, so I'm not requesting changes here. Just wanted to note that when available, the `Box` component is always preferred to be used over `Flex`.

Reply:
noted, in this version of Bitkit there is no `Box` available so I left as is